### PR TITLE
Test head pose

### PR DESCRIPTION
**Pull request by matemax**
Issue #29

### CODE REVIEW

File: tests/test_head_pose_estimation.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        cls.detection = TestHeadPose.detector.detectOne(cls.image, detect5Landmarks=True, detect68Landmarks=True)
+
+    def assertAngle(self, angle: float):
+        """
+        Assert degree measure of an angle
+
+        Args:
+            angle: -180 <= number <= 180
+        """
+        assert isinstance(angle, float), "bad angle instance"
```
Comment (Zeskbest) [inline tests/test_head_pose_estimation.py]: assertion message does not contain current instance type

Comment (Zeskbest) [reply tests/test_head_pose_estimation.py]: and below

Comment (Zeskbest) [reply tests/test_head_pose_estimation.py]: All these tests are very strange: you do not check anything but return types, is it needed?

File: tests/test_head_pose_estimation.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        angles = TestHeadPose.headPoseEstimator.estimateByBoundingBox(self.detection.boundingBox, self.image)
+        self.assertHeadPose(angles)
+
+    def test_estimate_head_pose_by_bounding_box_from_other_image(self):
+        """
+        Estimating head pose on image without faces by bounding box from other image.
+        """
+        image = VLImage.load(filename=NO_FACES)
+        angles = TestHeadPose.headPoseEstimator.estimateByBoundingBox(self.detection.boundingBox, image)
+        self.assertHeadPose(angles)
```
Comment (Zeskbest) [inline tests/test_head_pose_estimation.py:98]: Is there any angle-confidence variable to show angle fakeness?

Comment (matemax) [reply tests/test_head_pose_estimation.py:98]: I think that there is no  such confidence

File: tests/test_head_pose_estimation.py (hunk 1 above)
Comment (matemax) [reply tests/test_head_pose_estimation.py]: I check range of angle yet

Comment (matemax) [inline tests/test_head_pose_estimation.py]:
```suggestion
        assert isinstance(angle, float), f"bad angle instance, type of angle is {type(angle)}"
```

File: tests/test_head_pose_estimation.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        assert -180 <= angle <= 180
+
+    def assertHeadPose(self, headPose: HeadPose):
+        """
+        Assert head pose estimation.
+
+        Args:
+            headPose: an estimate head pose
+        """
+        assert isinstance(headPose, HeadPose), "bad head pose instance"
```
Comment (matemax) [inline tests/test_head_pose_estimation.py]:
```suggestion
        assert isinstance(headPose, HeadPose), f"bad head pose instance, type of head pose is {type(headPose)}"

```

File: tests/test_head_pose_estimation.py (hunk 1 above)
Comment (Zeskbest) [reply tests/test_head_pose_estimation.py]: OK

File: tests/test_head_pose_estimation.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        self.assertAngle(headPose.yaw)
+        self.assertAngle(headPose.roll)
+        self.assertAngle(headPose.pitch)
+
+        assert isinstance(headPose.getFrontalType(), FrontalType), "bad instance frontal type"
```
Comment (matemax) [inline tests/test_head_pose_estimation.py]:
```suggestion
        assert isinstance(headPose.getFrontalType(), FrontalType), f"bad instance frontal type, type of frontal type is {type(headPose.getFrontalType())}"

```

File: tests/test_head_pose_estimation.py [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+    def test_init_estimator(self):
+        """
+        Test init estimator.
+        """
+        estimator = TestHeadPose.faceEngine.createHeadPoseEstimator()
+        assert isinstance(estimator, HeadPoseEstimator), "bad estimator instance"
```
Comment (matemax) [inline tests/test_head_pose_estimation.py]:
```suggestion
        assert isinstance(estimator, HeadPoseEstimator), f"bad estimator instance, type of estimator  is {type(HeadPoseEstimator)}"
```

**Comment by Zeskbest** [comment]
ok

**Pull request merged**